### DISCUSSION TITLE
C# Language Specification for C# v8?

### DISCUSSION THREAD

**Discussion by Bartolomeus-649** [dotnet/csharplang#3303] [Q&A] [answered]
Where can you find the C# Language Specification for C# v8?

I found the official ecma specification of C# v5 at: https://www.ecma-international.org/publications/standards/Ecma-334.htm, and the C# v6 [specification here in this repo](https://github.com/dotnet/csharplang/tree/master/spec).
But now we are using C# v8 and it would be great to have the specifications for all the new stuff as well....even if were just the new things and/or changes from the previous version.

**Comment by 333fred** [comment, marked as answer]
You can find draft specs for features in the proposals folder. @BillWagner and @gafter are working on getting those specs vetted and integrated.
